DataTable.svelte

```diff
@@ -296,6 +314,12 @@
       <IconButton src={chevronLeftIcon} disabled={currentPageIndex === 0} onClick={toPrevPage} />
       <IconButton src={chevronRightIcon} disabled={currentPageIndex === lastPageIndex} onClick={toNextPage} />
       <IconButton src={pageLastIcon} disabled={currentPageIndex === lastPageIndex} onClick={toLastPage} />
+      <Select
+        values={options}
+        selected={pageSize.toString()}
+        reverse={true}
+        onChangeSelected={handleChangePagination}
+      />
```

Comment:
```suggestion
      <Select
        values={options}
        selected={pageSize.toString()}
        reverse
        onChangeSelected={handleChangePagination}
      />
```
If reverse is always true, you can write like this